api/src/main/resources/liquibase.xml

```diff
@@ -4,6 +4,345 @@
+        <createTable tableName="patientflags_tag">
+            <column autoIncrement="true" name="tag_id" type="int">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="name" type="VARCHAR(255)"/>
+            <column name="description" type="VARCHAR(1000)"/>
+            <column defaultValueNumeric="0" name="creator" type="int">
+                <constraints nullable="false"/>
+            </column>
+            <column name="date_created" type="datetime">
+                <constraints nullable="false"/>
+            </column>
+            <column name="changed_by" type="int"/>
+            <column name="date_changed" type="datetime"/>
+            <column defaultValueBoolean="false" name="retired" type="boolean">
+                <constraints nullable="false"/>
+            </column>
+            <column name="retired_by" type="int"/>
+            <column name="date_retired" type="datetime"/>
+            <column name="retire_reason" type="VARCHAR(255)"/>
+            <column name="uuid" type="CHAR(38)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet author="slubwama" id="patientflags-1554735240623-6">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="patientflags_tag_displaypoint"/>
+            </not>
+        </preConditions>
+        <comment>Create Table patientflags_tag_displaypoint</comment>
+        <createTable tableName="patientflags_tag_displaypoint">
+            <column name="tag_id" type="int">
+                <constraints nullable="false"/>
+            </column>
+            <column name="displaypoint_id" type="int">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet author="slubwama" id="patientflags-1554735240623-7">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="patientflags_tag_role"/>
+            </not>
+        </preConditions>
+        <comment>Create Table patientflags_tag_role</comment>
+        <createTable tableName="patientflags_tag_role">
+            <column name="tag_id" type="int">
+                <constraints nullable="false"/>
+            </column>
+            <column name="role" type="VARCHAR(50)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet author="slubwama" id="patientflags-1554735240623-11">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <indexExists indexName="displaypoint_id"/>
+            </not>
+        </preConditions>
+        <comment>Add a unique constraint displaypoint_id to table patientflags_tag_displaypoint on column displaypoint_id</comment>
+        <createIndex indexName="displaypoint_id" tableName="patientflags_tag_displaypoint">
```

Comment:
Why are you creating an index on the displaypoint_id column?

Reply:
@dkayiwa that was already there in SQLDIFF all I am trying to do is to have all that was there here in liquibase.

Reply:
Can you point me to where that was?

Reply:
@dkayiwa here  is the reference where this is created in the sqldiff https://github.com/openmrs/openmrs-module-patientflags/blob/7e59726ed24535ae97a1472d94de1c5c02a40082/api/src/main/resources/sqldiff.xml#L269

Reply:
What you are referencing is a foreign key. Not a unique index like the one you are creating.

Reply:
@dkayiwa  I have removed the creation of the Index for displaypoint_id column.

Reply:
It is not the only one, i see many others of the like. I just pointed it out as an example. 😊

Reply:
@dkayiwa  I have walked through all createIndex tags comparing them with the tables they are tagged to and it seems it was the only one. But if you Identify any kindly point it out here. thanks

Reply:
Have you looked at all places where you have createIndex?

Reply:
@dkayiwa yes